index.css

```diff
@@ -112,19 +118,253 @@ nav {
   margin-top: 110px;
- }
- .myresent img {
-   background-color: #1c1a19;
-   margin-top: 110px;
- }
+}
+
+.contenido {
+  display: grid;
+  row-gap: 30px;
+  justify-content: center;
+  align-items: center;
+  margin-top: 90px;
+}
+
+.box {
+  background-image: url(../Pictures/profesionalart.png);
+  background-repeat: no-repeat;
+}
+
+.multi {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  color: #091e42;
+}
+
+.daily {
+  width: 327px;
+  height: 147px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.botones4 {
+  width: 80px;
+  height: 40px;
+  margin-left: 66px;
+}
+
+.botones4 ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  font-size: 0;
+  align-items: center;
+  width: 49px;
+}
+
+.botones4 button {
+  background-color: rgba(255, 255, 255, 0.24);
+  color: #091e42;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  margin-left: 8px;
+  height: 49px;
+}
+
+.see:hover {
+  background-color: rgb(255, 140, 0);
+}
+
+.proyect button {
+  width: 124px;
+  height: 48px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  background: #ff6b00;
+  color: #fff;
+  margin-top: 20px;
+}
+
+.dashbord {
+  width: 295px;
+  height: 81px;
+  padding-top: 50px;
+  padding-left: 20px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.texto3 {
+  width: 295px;
+  height: 87px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  padding-left: 20px;
+}
+
+.botones3 button {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  background-color: rgba(255, 255, 255, 0.24);
+  color: #fff;
+  margin-right: 10px;
+  margin-top: 10px;
+  border: none;
+}
+
+.botones3 ul {
+  width: 61px;
+  height: 40px;
+  list-style: none;
+  display: flex;
+  padding: 10px 12px 0 20px;
+}
+
+.brian button {
+  width: 327px;
+  height: 60px;
+  margin-top: 15px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  background-color: #ff6b00;
+  font-size: 17px;
+  color: #fff;
+}
+
+.about {
+  background: #1c1a19 url(pictures/Frame1.png) 100% 95% no-repeat;
+  margin-top: 80px;
+  height: 645px;
+}
+
+.about h3 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 72px;
+  line-height: 88px;
+  color: #ff6b00;
+  padding-top: 90px;
+  text-align: center;
+}
+
+.about p {
+  width: 327px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  color: #f4f5f7;
+  margin-left: 20px;
+}
```

Comment:
Kindly increase the font size in the about section so it matches the Figma design  : 👍  😃👍

Your Design :
![image](https://user-images.githubusercontent.com/87448628/159065436-5b75896d-02fd-4420-aa52-8e0ccaa4e323.png)

Figma Design : 
![image](https://user-images.githubusercontent.com/87448628/159065721-63a504a1-4059-4411-89b0-68c08cf94500.png)

Reply:
@[Danieldotcomcoder](https://github.com/Danieldotcomcoder) hello coach got a question the dimencion are375 x696
why it looks like that

Reply:
i was checking on figma and it shows the fond-size 16px and when I place the dimension 375 x695 I see it good

Reply:
im going to make the changes